node/src/gql/query/me/category-answers.ts

```diff
@@ -0,0 +1,62 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+
+import { Types } from 'mongoose';
+import { User } from 'models/User';
+import { Mentor as MentorModel } from 'models';
+import { GraphQLList, GraphQLObjectType, GraphQLString } from 'graphql';
+import { Status } from 'models/Answer';
+
+const response = new GraphQLObjectType({
+  name: 'response',
+  fields: {
+    answerText: { type: GraphQLString },
+    questionText: { type: GraphQLString },
+  },
+});
+
+export const categoryAnswers = {
+  type: GraphQLList(response),
+  args: {
+    category: { type: GraphQLString },
+  },
+  resolve: async (_: any, args: any, context: { user: User }) => {
```

Comment:
guessing the `any` types here are accounting for one or two of the new lint issues?

For sure, `args` can be explicitly typed as

```typescript
args: { category: string }
```

...?

I remember from past attempts that the `parent` args, which you have named as `_` might be tricky to type?

If that's the case, let's a sharable/importable type somewhere under gql, and then--if it can't be strictly typed--just ignore the type errors one time there, e.g.

```
// maybe it works like this?
export type GqlResolverParentAny = Record<string, unknown>

// or maybe needs to be something like this?
// eslint-disable-next-line <rule about no any>
export type GqlResolverParentAny = any
```